test/test_router.js

```diff
@@ -112,5 +114,23 @@ describe('Router test', () => {
       });
     });
   });
+
+  describe('endpoint urls', () => {
+    const preRoute = 'Observations(1)/';
+    Object.keys(entities).forEach(entityName => {
+      it('should acces to the property of a singular association', (done) => {
```

Comment:
and since you need to change the line... remove the parenthesis around `done`, please :)